deployment/charts/cluster-connect-gateway/values.yaml

```diff
@@ -186,7 +186,9 @@ controller:
   logLevel: 2
 
   # Additional command line flags to pass.
-  extraArgs: []
+  extraArgs: [
+    "--connection-probe-timeout=5m",
```

Comment:
Consider giving connectionProbeTimeout its own Helm value rather than sticking this option in extraArgs[]

Reply:
done here https://github.com/open-edge-platform/cluster-connect-gateway/pull/41/commits/93d475d3ebc482ee0919a12d9ab9642d6cd71ffc